ResearchKit/Common/ORKTaskViewController.h

```diff
@@ -254,7 +257,7 @@ task view controller and pass that data to `initWithTask:restorationData:` when
  objects in the result hierarchy.
  */
 ORK_CLASS_AVAILABLE
-@interface ORKTaskViewController : UIViewController <ORKStepViewControllerDelegate, UIViewControllerRestoration>
+@interface ORKTaskViewController : UIViewController <ORKStepViewControllerDelegate, ORKReviewStepViewControllerDelegate, UIViewControllerRestoration>
```

Comment:
Move the delegate declaration into internal, since `ORKReviewStepViewControllerDelegate` is not part of public API